Sources/XiEditor/Annotation.swift

```diff
@@ -0,0 +1,125 @@
+// Copyright 2018 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+/// Supported annotation types.
+enum AnnotationType: String, CaseIterable {
+    case Selection = "selection"
+    case Find = "find"
```

Comment:
suggestion:

if you do `case selection`, it will be same as `case selection = "selection"`.

(and camelCase is more swifty)

what do you think?